system/addon-manifest.xml

```diff
@@ -29,6 +29,7 @@
   <addon>screensaver.xbmc.builtin.black</addon>
   <addon>screensaver.xbmc.builtin.dim</addon>
   <addon>script.module.pil</addon>
+  <addon>script.module.pycryptodome</addon>
```

Comment:
i thought the manifest file decides which addons are enabled by default?
iirc if it's not listed the script.module will be installed, but disabled.